app/Http/Controllers/AuthController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Hash;
+use Session;
+use App\Models\User;
+use Illuminate\Support\Facades\Auth;
+
+class AuthController extends Controller
+{
+
+    public function index()
+    {
+        return view('auth.login');
+    }  
+      
+
+    public function customLogin(Request $request)
+    {
+        $request->validate([
+            'email' => 'required',
+            'password' => 'required',
+        ]);
+   
+        $credentials = $request->only('email', 'password');
+        if (Auth::attempt($credentials)) {
+            return redirect()->intended('dashboard')
+                        ->withSuccess('Signed in');
+        }
+  
+        return redirect("login")->withSuccess('Login details are not valid');
+    }
+
+
+
+    public function registration()
+    {
+        return view('auth.registration');
```

Comment:
ini kyknya register.blade.php deh namanya
```suggestion
        return view('auth.register');
```